GVFS/GVFS/CommandLine/DiagnoseVerb.cs

```diff
@@ -113,6 +113,14 @@ protected override void Execute(GVFSEnlistment enlistment)
                             this.ServiceName,
                             copySubFolders: true);
 
+                        // upgrader
+                        this.CopyAllFiles(

```

Comment:
Do we need to copy all of these files? How big can they get? Would a directory listing be good enough, and avoid adding unnecessary size and bulk to diagnose?

Reply:
Max size I've seen of a gvfs_*.log file is 4KB. Git and GVFS installer logs reach 2MB and 30KB respectively. Also it gets generated only when a user runs upgrade.

Reply:
@alameenshah fine to do this in a future PR, but I think a directory listing would give us all the info we need, and we already have code to do that. There's no need to copy the installers themselves into the zip file.

Reply:
>>There's no need to copy the installers themselves into the zip file.

Oh. We Don't copy the installers at all. Only the log files.

Reply:
Ah yea I missed that it was just the logs, I thought it was the installers themselves.

In that case, I think we _also_ need a directory listing of the downloaded installers, since that is what drives the hooks to show an upgrade reminder to the user. When someone complains about that, we need to have a way to diagnose what's happening. So please add that as well.